src/server.ts

```diff
@@ -29,8 +84,10 @@ function getRovers(expressRequest: express.Request, expressResponse: express.Res
 function initRouter(): express.Router {
     const router = express.Router();
 
-    router.get('/test', getTest);
-    router.get('/rovers', getRovers);
+    router.get("/test", getTest);
+    router.get("/rovers", getRovers);
+    router.get("/rovers/photos", getRoversPhotos);
+    router.get("/rovers/*/photos/*", getRoversPhotosGeneric);
```

Comment:
Alright, time to start thinking about route parameters!

https://expressjs.com/en/guide/routing.html#route-parameters